README.md

```diff
@@ -36,17 +33,17 @@ spee.ch is a single-serving site that reads and publishes images and videos to a
 
 #### POST
 * /api/publish
-  * example: `curl -X POST -F 'name=MyPictureName' -F 'nsfw=false' -F 'file=@/path/to/my/picture.jpeg' https://spee.ch/api/publish`
```

Comment:
@lyoshenka I gather best practices re: API documentation have evolved. Should @billbitt consider swagger or something else to document these calls?

(May be out of scope for this PR and should instead be opened separately)

Reply:
I'll discuss with @lyoshenka and open separate PR for needed changes